modules/pam_listfile/pam_listfile.c

```diff
@@ -181,7 +181,7 @@ pam_sm_authenticate (pam_handle_t *pamh, int flags UNUSED,
 	}
     }
 
-    /* Short-circuit - test if this session apply for this user */
+    /* Short-circuit - test if this session applies for this user */
```

Comment:
Ideally, the commit message should mention that the typo being fixed is in a comment.

Reply:
Adjusted